stack.yaml

```diff
@@ -13,8 +15,34 @@ packages:
 - servant-pipes/
 - servant-swagger/
 
-# allow-newer: true # ignores all bounds, that's a sledgehammer
-# - doc/tutorial/
-
 extra-deps:
-- hspec-wai-0.10.1
+- aeson-2.0.2.0
+- attoparsec-0.14.2
+- base-compat-0.12.1
+- base-compat-batteries-0.12.1
+- github: yesodweb/wai
+  commit: 02441350b50558f628158ab50b023b5328f38a41
+  subdirs:
+    - warp
```

Comment:
This should be unnecessary since https://github.com/yesodweb/wai/commit/493610faeb7c2dedd28ecb43e6a6adfb6e0f4eac was released.